packages/ffe-form/less/theme.less

```diff
@@ -16,6 +16,8 @@
     --ffe-v-radio-button-label-color: var(--ffe-g-primary-color);
     --ffe-v-tooltip-icon-color: var(--ffe-g-primary-color);
     --ffe-v-tooltip-border-color: var(--ffe-g-border-color);
+    --ffe-v-message-font-size: var(--ffe-fontsize-body-text);
+    --ffe-v-message-line-heigth: 1.5rem;
```

Comment:
Jag kan ikke ha uniit less her fordi att det ikke funker på feks en padding
` padding: 0 0 calc(var(--ffe-v-message-line-heigth) + var(--ffe-spacing-xs));`

Reply:
Trenger `line-height` og `padding` å være det samme, egentlig? Ser for meg at line-height definert i rem kan være litt uforutsigbart.

Reply:
Paddingen er der for å reservere plass til en enlinjers feilmelding. De må være like for at innholdet ikke skal hoppe.

Jeg foretrekker å bruke `line-height: 1.5` på alt, men det er brukt `rem` i hele [typography.less](https://github.com/SpareBank1/designsystem/blob/3a4d6b2e14b8aa93318abdb176b41e448af5a31e/packages/ffe-core/less/typography.less).

Reply:
@antidecaf utfordringen att bruke noe annet en line-height we ju text zoom. Det vill da se greit ut før fler tenker jag. Annars undgår man kun hoppet med standard zoom. 
Man vill ju oansett få hopp hvis man har lange meldninger.

Reply:
@a504aa1 her kan vi ikke bruke 1.5 uten rem(hvis vi skall bruke den samme custom propertien før paddingen).. fordi hvis vi gjør det blir det 

```
padding: 1.5; // istellet for feks padding: 1.5rem; 
```

Det skjønner browsers dålig.

Reply:
@pethel Mente ikke at du skulle bruke `line-height: 1.5` her. Tenkte mer generelt at jeg synes det er enklere å sette `line-height: 1.5` på alt enn å sette `line-height: Xrem` for alle forskjellige tekststørrelser slik det er gjort i designsystemet. Da ville f.eks. linjehøyde alltid vært likt `1.5em`.

Reply:
Verdien kan selvsagt være det samme, jeg mente bare om custom propertyen trenger å gjenbrukes i dette tilfellet. Enig i at `rem` ikke er den mest hensiktsmessige uniten for `line-height` sånn på generelt grunnlag, men ser ut til at det er et litt lengre lerret å bleke. :)